examples/cloudify-hello-world-example/openstack.yaml

```diff
@@ -2,6 +2,7 @@ tosca_definitions_version: cloudify_dsl_1_3
 
```

Comment:
Truth is, I think we need a submodule to this: https://github.com/cloudify-community/blueprint-examples/tree/master/hello-world-example

Reply:
You are right about this, do you think we should wait until release the plugin or we can do it now